### PR TITLE
testing/cage: new aport

### PR DESCRIPTION
**Pull request by durrendal**
Create a new aport for cage, a Wayland kiosk environment.

### CODE REVIEW

File: testing/cage/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+pkgname=cage
+pkgver=0.1.1
+pkgrel=0
+pkgdesc="A Wayland Kiosk"
+url="https://github.com/Hjdskes/cage"
+arch="x86_64"
```
Comment (maxice8) [inline testing/cage/APKBUILD]: Is it really x86_64 only ?

File: testing/cage/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+license="MIT"
+depends="wlroots wayland libxkbcommon xorg-server-xwayland"
```
Comment (maxice8) [inline testing/cage/APKBUILD]: only the last one should be here, the rest are detected via linked libraries

File: testing/cage/APKBUILD (hunk 1 above)
Comment (durrendal) [reply testing/cage/APKBUILD]: Right you are, looks like it builds on all arches.

File: testing/cage/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+options="!check" #No checks
+source="$pkgname"-"$pkgver".tar.gz::https://github.com/Hjdskes/cage/archive/v"$pkgver".tar.gz
+
+prepare () {
+	meson --buildtype=release -Dxwayland=true --prefix /usr "$srcdir/build"
+}
+	
+build () {
+	ninja -C "$srcdir/build"
+}
```
Comment (jjsearle) [inline testing/cage/APKBUILD]: > IC:[AL8]:APKBUILD:21:trailing whitespace

File: testing/cage/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+pkgdesc="A Wayland Kiosk"
+url="https://github.com/Hjdskes/cage"
+arch="all"
+license="MIT"
+depends="xorg-server-xwayland"
+makedepends="meson cmake wlroots-dev wayland-protocols"
+options="!check" #No checks
+source="$pkgname"-"$pkgver".tar.gz::https://github.com/Hjdskes/cage/archive/v"$pkgver".tar.gz
+
+prepare () {
```
Comment (jjsearle) [inline testing/cage/APKBUILD]: > TC:[AL10]:APKBUILD:15:do not use space before function parenthesis

File: testing/cage/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+	meson --buildtype=release -Dxwayland=true --prefix /usr "$srcdir/build"
+}
+	
+build () {
```
Comment (jjsearle) [inline testing/cage/APKBUILD]: > TC:[AL10]:APKBUILD:19:do not use space before function parenthesis

File: testing/cage/APKBUILD (hunk 3 above)
Comment (jjsearle) [inline testing/cage/APKBUILD]: >IC:[AL8]:APKBUILD:21:trailing whitespace
TC:[AL10]:APKBUILD:15:do not use space before function parenthesis
TC:[AL10]:APKBUILD:19:do not use space before function parenthesis

Couple of linting issues.

Comment (durrendal) [reply testing/cage/APKBUILD]: I think these should all be taken care of now

File: testing/cage/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+depends="xorg-server-xwayland"
+makedepends="meson cmake wlroots-dev wayland-protocols"
+options="!check" #No checks
+source="$pkgname"-"$pkgver".tar.gz::https://github.com/Hjdskes/cage/archive/v"$pkgver".tar.gz
+
+prepare(){
+	meson --buildtype=release -Dxwayland=true --prefix /usr "$srcdir/build"
+}
+	
+build(){
```
Comment (maxice8) [inline testing/cage/APKBUILD]: need space between () and {

File: testing/cage/APKBUILD [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+url="https://github.com/Hjdskes/cage"
+arch="all"
+license="MIT"
+depends="xorg-server-xwayland"
+makedepends="meson cmake wlroots-dev wayland-protocols"
+options="!check" #No checks
+source="$pkgname"-"$pkgver".tar.gz::https://github.com/Hjdskes/cage/archive/v"$pkgver".tar.gz
+
+prepare(){
+	meson --buildtype=release -Dxwayland=true --prefix /usr "$srcdir/build"
```
Comment (maxice8) [inline testing/cage/APKBUILD]: isn't it better to do it like most other APKBUILDs that use meson ?

**Pull request closed without merging**